src/yunohost/domain.py

```diff
@@ -273,7 +273,7 @@ def domain_dns_conf(domain, ttl=None):
             '(?=.*(;[\s]*|")p=(?P<p>[^";]+))'), dkim_content, re.M|re.S
         )
         if dkim:
-            result += '\n{host}. {ttl} IN TXT "v={v}; k={k}; p={p}"'.format(
+            result += '\n{host} {ttl} IN TXT "v={v}; k={k}; p={p}"'.format(
```

Comment:
Here's 2 copy/paste:

- From Gandi
```
mail._domainkey 1800 IN TXT "v=DKIM1; k=rsa; p=XXXX"
_dmarc 1800 IN TXT "v=DMARC1; p=none"
```

- From BookMyName
```
mail._domainkey           900  TXT    "v=DKIM1; k=rsa; p=XXX"
_dmarc                    900  TXT    "v=DMARC1; p=none"
```

The only difference is the keyword `IN` (which is 'record class', always IN for INternet), but syntax is the same, only subdomain without trailing period.

Quote from wikipedia : "host names that do not end in a period are relative to the zone origin" (https://en.wikipedia.org/wiki/Zone_file#File_format)

To be really nerdy, we can read corresponding RFC: https://tools.ietf.org/html/rfc1034#page-8